packages/app/src/runner/studio/AssertionOptions.ce.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <div
+    ref="popper"
+    class="assertion-options"
+    :style="{ position: 'absolute' }"
+  >
+    <div
+      v-for="{ name, value } in options"
+      :key="`${name}${value}`"
+      class="assertion-option"
+      @click.stop="() => onClick(name, value)"
```

Comment:
I think you can just pass the function in like this
```suggestion
      @click.stop="onClick(name, value)"
```